FND/TransactionsExt/REATransactions.rdf

```diff
@@ -105,6 +105,19 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">In REA, transactions may include those which are not between legal entities,such as for example internal transactions within a business and between business units. This term Contractual Transaction Party forms the basis for all party definitions for transactions which have some formal contractual basis as being between discrete legal entities (legal persons or other contractually capable entities e.g. non incorporated entities). This is the basis for all derivatives transactions, securities market transactions and so on.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-fnd-txn-rea;CoveredTransaction">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-txn-rea;ContractualTransaction"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isGovernedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-agr-ctr;MasterAgreement"/>
```

Comment:
@rivettp it was in the ontology we are eliminating - I didn't want to lose that, since I think it "is a thing", but the whole transaction-related stuff needs to be revamped.  The only point was to move it over so that we didn't lose the idea, allowing us to delete the other informative ontology.

Reply:
@rivettp After this change we will have only 3 remaining informative ontologies, all in TransactionsExt, FYI